user/models.go

```diff
@@ -87,6 +87,16 @@ type User struct {
 	PasswordResets    []*UserPasswordReset    `bson:"password_resets"`
 }
 
+// UserPresence contains details about a user online status
+type UserPresence struct {
```

Comment:
You've not completed how this works with the user object. A struct alone really does nothing.

Reply:
Yes please
I'm about to fix an issue with that